public/gallery.html

```diff
@@ -106,12 +132,180 @@
 			<button class="bg-black text-white text-lg ring-2 py-1 px-4 rounded-lg self-center" type="submit"><strong>Apply</strong></button>
 		</div>
 	</div>
-<!-- End filters container -->
 </div>
+<!-- End filters container -->
+
 <!-- Menu end -->
-<div>
-	<img  src="Assets/images/gallery/percare_gallery_hero.png" alt="Animals posing in a art gallery">
+
+<!-- Grid section -->
+<div id="gridBackground">
```

Comment:
Name is tied to CSS property. What happens if we decide that grid is not a good option and we need to change to flex?